editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -620,6 +620,12 @@ impl Fsm for PathToolFsmState {
 
 						if let Some(closest_segment) = &tool_data.segment {
 							overlay_context.manipulator_anchor(closest_segment.closest_point_to_viewport(), false, Some(COLOR_OVERLAY_YELLOW));
+							if let (Some(handle1), Some(handle2)) = closest_segment.handle_positions(document.metadata()) {
+								overlay_context.line(closest_segment.closest_point_to_viewport(), handle1, Some(COLOR_OVERLAY_YELLOW));
+								overlay_context.line(closest_segment.closest_point_to_viewport(), handle2, Some(COLOR_OVERLAY_YELLOW));
+								overlay_context.manipulator_handle(handle1, false, Some(COLOR_OVERLAY_YELLOW));
+								overlay_context.manipulator_handle(handle2, false, Some(COLOR_OVERLAY_YELLOW));
+							}
```

Comment:
This should probably be split up in case one handle exists but not the other, so they aren't dependent on each other both existing to draw either one of them.